var/spack/repos/builtin/packages/py-scikit-learn/package.py

```diff
@@ -86,8 +89,9 @@ class PyScikitLearn(PythonPackage):
     depends_on("py-matplotlib@3.1.3:", type="test")
     depends_on("py-scikit-image@0.16.2:", type="test")
     depends_on("py-pandas@1.0.5:", type="test")
-    depends_on("py-pytest@5.3.1:", type="test")
+    depends_on("py-pytest@7.1.2:", type="test")
     depends_on("py-pyamg@4:", type="test")
+    depends_on("py-pooch@1.6:", type="test")
```

Comment:
Why are there only some of the test dependencies listed here? In [_min_dependencies.py](https://github.com/scikit-learn/scikit-learn/blob/1.3.0/sklearn/_min_dependencies.py) there are additional ones listed: py-pytest-cov, py-ruff, py-black, py-mypy, py-numpydoc

Reply:
I assume those are style tests, not ones run by pytest.